.dart_tool/package_config.json

```diff
@@ -243,7 +333,7 @@
     },
     {
       "name": "test",
-      "rootUri": "file:///Users/parthbasole/.pub-cache/hosted/pub.dev/test-1.25.1",
+      "rootUri": "file:///Users/parthbasole/.pub-cache/hosted/pub.dev/test-1.25.2",
```

Comment:
Why parthbasole? This is hardcoded?

Reply:
Resolved it by adding the folder in .gitignore